platform_umbrella/apps/control_server_web/lib/control_server_web/live/postgres/show.ex

```diff
@@ -229,6 +241,7 @@ defmodule ControlServerWeb.Live.PostgresShow do
         >
           Edit Versions
         </:tab>
+        <:tab selected={@live_action == :backups} patch={backups_url(@cluster)}>Backups</:tab>
```

Comment:
Can we hide this link if there are no backups available

Reply:
Not sure the juice was worth the squeeze but it's hidden now:
https://github.com/batteries-included/batteries-included/pull/1719/commits/92a4a0936cbdc00eeefe706fd752d5ce3422e0b1